rest-api/rest-api-quiz.md

```diff
@@ -223,26 +223,19 @@
 - [x] common use cases
 - [ ] your tech stack
 
-#### Q34. What is the purpose of an OAuth refresh token?
+#### Q38. What is the purpose of an OAuth refresh token?
 - [ ] to share user profile information
 - [ ] to update an API configuration
 - [ ] to keep a web session active
 - [x] to retrive an access token
 
-#### Q35. What is Time to First Hello World?
+#### Q39. What is Time to First Hello World?
 - [x] how long it takes for a developer to do something with your API
 - [ ] how long it takes to start a new programming language
 - [ ] how long it takes to install your SDK
 - [ ] how long it takes to read your documentation
 
-#### Q35. What is Time to First Hello World?
-- [x] how long it takes for a developer to do something with your API
-- [ ] how long it takes to start a new programming language
-- [ ] how long it takes to install your SDK
-- [ ] how long it takes to read your documentation
-
```

Comment:
Hello, @LuigiZambetti! Thanks for raising this PR looks good, but I think you accidentally removed the answers, please double-check and revert it back. Let me know if you need any help from me.

Reply:
That question was doubled. I've just remove one of both.

Reply:
ok, agree

```
#### Q35. What is Time to First Hello World?
- [x] how long it takes for a developer to do something with your API
- [ ] how long it takes to start a new programming language
- [ ] how long it takes to install your SDK
- [ ] how long it takes to read your documentation

#### Q35. What is Time to First Hello World?
- [x] how long it takes for a developer to do something with your API
- [ ] how long it takes to start a new programming language
- [ ] how long it takes to install your SDK
- [ ] how long it takes to read your documentation
```